data-access-analysis-tools/nb-intro.rst

```diff
@@ -168,6 +168,27 @@ The read-only "notebooks/tutorial-notebooks" directory will *always* contain the
 Notebooks can be edited and executed in this directory, but **changes cannot be saved to this directory**.
 Users wishing to edit, execute, *and save* versions of these notebooks should copy them to a different path in their home directory.
 
+For example, to obtain a copy of the DP0.2 introductory notebook in the home directory which can be 
+opened, edited, and saved, execute the following commands in a terminal to copy the file and
+change its permissions to allow user write access.
+The dollar signs indicate terminal command line executable statements.
+Lines without dollar signs indicate standard output.
+
+.. code-block:: bash
+
+      $ cd ~
+      $ cp notebooks/tutorial-notebooks/DP02_01_Introduction_to_DP02.ipynb .
+      $ ls DP02*
+      DP02_01_Introduction_to_DP02.ipynb
+      $ ls -lah DP02_01_Introduction_to_DP02.ipynb 
+      -r--r--r-- 1 melissagraham melissagraham 37K Nov 13 21:14 DP02_01_Introduction_to_DP02.ipynb
+      $ chmod u+w DP02_01_Introduction_to_DP02.ipynb 
+      $ ls -lah DP02_01_Introduction_to_DP02.ipynb 
+      -rw-r--r-- 1 melissagraham melissagraham 37K Nov 13 21:14 DP02_01_Introduction_to_DP02.ipynb
```

Comment:
Do we want to add more explanatory material here, or just leave it as the commands with no explanation? (e.g., we could explain that "cd ~" changes to your home directory, "chmod u+w..." changes permissions to make it writeable, etc.) Feel free to ignore this comment if that makes things too busy/complicated.

Reply:
Added a paragraph with a step-by-step explanation -- I think you're right, many people have no command-line experience yet. May as well use as a teachable moment!